src/components/ChangePlan/PlanCalculator.test.js

```diff
@@ -0,0 +1,374 @@
+import React from 'react';
+import { render, cleanup, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import PlanCalculator from './PlanCalculator';
+import { IntlProvider } from 'react-intl';
+import { AppServicesProvider } from '../../services/pure-di';
+import { BrowserRouter } from 'react-router-dom';
+
+describe('PlanCalculator component', () => {
+  afterEach(cleanup);
+
+  const dependencies = {
+    appSessionRef: {
+      current: {
+        userData: {
+          user: {
+            plan: {},
+          },
+        },
+      },
+    },
+  };
+
+  it('should show all plan types in tabs', async () => {
```

Comment:
How about validate for the slider to have some elements? Or the slider to be present here.